.travis.yml

```diff
@@ -29,26 +29,11 @@ android:
     # Latest artifacts in local repository
     - extra-google-m2repository
     - extra-android-m2repository
-    - android-sdk-license-.+
-    - '.+'
     # Specify at least one system image
-    - sys-img-armeabi-v7a-google_apis-$ANDROID_API_LEVEL
-    - sys-img-armeabi-v7a-android-$ANDROID_API_LEVEL_22
-
-# list of directories to Cache
-
-before_install:
```

Comment:
This is a [work around for api 25](https://github.com/danielpassos/fh-android-sdk/commit/336fe5e0a942fa2d3411e48df89555cab06513b6)

Reply:
`26.0.0-beta2` might be using the same license as api 25 and I guess that's why the build worked until travis build#40